docs/source/guides/integrations.mdx

```diff
@@ -0,0 +1,170 @@
+# Integrate any ML framework with the Hub
+
+The Huggingface Hub makes it easy to host and share models with the community. It already has support for
+[dozens of libraries](https://huggingface.co/docs/hub/models-libraries) in the Open Source ecosystem. We are always
+working on expanding this support to push Machine Learning forward. The `huggingface_hub` library plays a key role in
+this process, allowing any Python script to easily push and load files.
+
+When talking about integrating a library to the Hub, we have 4 main topics:
```

Comment:
Maybe it would be nice to mention the first two are essential as I think inference support is often cherry on top.